docs/detections/view-osquery-results.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[view-osquery-results]]
+=== Examine Osquery results
+Osquery provides you with relevant, timely data that you can use to better understand and monitor your environment. When you run queries, results are indexed and then displayed the Results table, which you can filter, sort, and interact with.
```

Comment:
```suggestion
Osquery provides relevant, timely data that you can use to better understand and monitor your environment. When you run queries, results are indexed and displayed the Results table, which you can filter, sort, and interact with.
```
Also a little skeptical of "relevant" since that seems subjective and maybe promising too much. "Timely" is also a relative concept, but I think it's OK here.